pandas/tests/generic/test_series.py

```diff
@@ -227,3 +227,8 @@ def test_valid_deprecated(self):
         # GH18800
         with tm.assert_produces_warning(FutureWarning):
             pd.Series([]).valid()
+
+    def test_axis_supported(self):
+        # Supporting axis for compatibility, detailed in GH-18589
+        s = Series(range(5))
+        s.rename({}, axis=0)
```

Comment:
test with axis > 0, which will raise

Reply:
move to pandas/tests/series/test_alter_axes.py with other rename tests.

Reply:
Moved tests, can you clarify on the comment about axis > 0 raising?
I added a test but it doesn't raise, we're popping it and never using it - did you mean that it should raise an error? I can implement if that's intended.

Reply:
there should a call of
_getaxis_number 
somewhere if not that’s an error

Reply:
There's no call to that here, as we're not using the argument anywhere.
Would you still like me to add this (and test that the addition is working as intended) despite the argument not being used?